cpp/daal/include/algorithms/decision_forest/decision_forest_classification_model_builder.h

```diff
@@ -112,7 +112,8 @@ class DAAL_EXPORT ModelBuilder
     NodeId addLeafNode(const TreeId treeId, const NodeId parentId, const size_t position, const size_t classLabel)
     {
         NodeId resId;
-        _status |= addLeafNodeInternal(treeId, parentId, position, classLabel, resId);
+        const double cover = 0.0; // TODO: Add cover
```

Comment:
Cover is needed for treeShap. We are using it in regression, not in in classification. This will be added. For now it's just there to have symmetric classification & regression APIs